Sources/UIApplication+handleSDLEvents.swift

```diff
@@ -236,6 +236,12 @@ public func onNativeTouch(
     pressure: JavaFloat,
     timestampMs: JavaLong
 ) {
+    if env == nil || view == nil {
+        // We're in an invalid state where the JNI has exploded somehow
+        // Passing anything on to UIKit now would probably lead to a crash
+        return
+    }
```

Comment:
@michaelknoch FYI – I haven't tested this but if you have a device and a working build you could test that it works in the standard case and if so I have a good feeling we might solve most of the UIKit crashes with it